.babelrc

```diff
@@ -1,7 +1,16 @@
 {
-    "presets": ["./.babelrc.js"],
+    "presets": ["env", "react", "es2017", "es2015", "./.babelrc.js"],
     "ignore": [
         "./index.js"
+    ],
+    "plugins": [
```

Comment:
Well we don't really need the plugins and presets here since these are covered by `.babelrc.js`.